cypress/e2e/models/administration/jira-connection/jira.ts

```diff
@@ -242,6 +250,12 @@ export class Jira {
         };
     }
 
+    static ApplyFilterByName(value: string) {
```

Comment:
According to most conventions. static methods should start in lowercase.

https://google.github.io/styleguide/tsguide.html#static-this

https://github.com/airbnb/javascript#classes--methods-use-this

Reply:
Fixed